pkgs/development/python-modules/pytest-server-fixtures/default.nix

```diff
@@ -11,8 +11,8 @@ buildPythonPackage rec {
     sha256 = "c89f9532f62cf851489082ece1ec692b6ed5b0f88f20823bea25e2a963ebee8f";
   };
 
-  buildInputs = [ pytest ];
-  propagatedBuildInputs = [ setuptools-git pytest-shutil pytest-fixture-config psutil requests future ];
+  buildInputs = [ pytest_3 ];
```

Comment:
Maybe just add this as a patch https://github.com/manahl/pytest-plugins/commit/abffd6727ed56fba3b9154a6254d29191d8eba4b ?

Reply:
I think that's more of a hassle than it's worth because we'd have to vendor the patch.

Reply:
Right, though they did say there would be a release soon https://github.com/manahl/pytest-plugins/issues/125#issuecomment-474980815.

This should be fine for now.